tests/test_command_bridge_channel_validation.py

```
@@ -0,0 +1,62 @@
+import asyncio
+import os
+import sys
+import unittest
+from unittest.mock import MagicMock, patch
+
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", "src"))
+
+from mmrelay.matrix_utils import _get_meshtastic_interface_and_channel
+
+
+def _run(coro):
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `_run`

(ANN202)